certification/engine/engine_test.go

```diff
@@ -0,0 +1,56 @@
+package engine
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/runtime"
+)
+
+var _ = Describe("Engine Creation", func() {
+	Describe("When getting a new engine for a configuration", func() {
+		Context("with a valid configuration", func() {
+			cfg := runtime.Config{
+				Image:          "dummy/image",
+				EnabledChecks:  ContainerPolicy(),
+				ResponseFormat: "json",
+				LogFile:        "some.log",
+			}
+
+			It("should return and engine and no error", func() {
+				engine, err := NewForConfig(cfg)
+				Expect(err).To(BeNil())
```

Comment:
A bit nitpicky... I prefer:

Expect(err).ToNot(HaveOccurred())

It basically does the same thing, but hides the nil part?

Reply:
Fixed!